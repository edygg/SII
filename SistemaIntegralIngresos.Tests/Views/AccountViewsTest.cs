using System;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using ApprovalTests.Asp.Mvc;
using SistemaIntegralIngresos.Controllers;
using ApprovalTests.Reporters;

namespace SistemaIntegralIngresos.Tests.Views
{
    [TestClass]
    [UseReporter(typeof(FileLauncherReporter))]
    public class AccountViewsTest : MvcTest
    {
        [TestMethod]
        public void RegisterAccountViewTest()
        {
            MvcApprovals.VerifyMvcPage(new AccountController().Register);
        }

    }
}
