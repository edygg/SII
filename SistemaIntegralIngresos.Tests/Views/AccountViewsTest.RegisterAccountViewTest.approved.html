<!DOCTYPE html>
<html lang="en">
    <head><base href="http://localhost:19610/Account">
        <meta charset="utf-8" />
        <title>Usuarios | SII</title>
        <link href="/favicon.ico" rel="shortcut icon" type="image/x-icon" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
        <link href="/Content/foundation.min.css" rel="stylesheet"/>

        <link href="/Content/Layout.css" rel="stylesheet"/>

        <link href="/Content/foundation-icons/foundation-icons.css?v=-6_zzR_BwLqFqCPKpXxa56eDHYlTBHOUql5yYn8-ikw1" rel="stylesheet"/>

        <script src="/Scripts/modernizr-2.6.2.js"></script>

    </head>
    <body>
        <header>
            <nav class="top-bar" data-topbar role="navigation">
    <ul class="title-area">
        <li class="name">
            <h1><a href="/">Inicio</a></h1>
        </li>
        <li class="toggle-topbar menu-icon"><a href="#"><span>Menu</span></a></li>
    </ul>

    <section class="top-bar-section">
        <!-- Right Nav Section -->
        <ul class="right">
        </ul>
    </section>
</nav>
        </header>
        <section id="body">
            <header>
                <div class="row">
    <div class="medium-10 small-9 columns text-center">
        <h1>Sistema Integral de Ingresos (<acronym title="Sistema Integral de Ingresos">SII</acronym>)</h1>
        <h2>Universidad Tecnológica Centroamericana</h2>
    </div>
    <div class="medium-2 small-3 columns">
        <img src="/Images/unitec-logo.png" alt="Unitec Logo" />
    </div>
</div>
<br />
<div class="row">
    <div class="text-center">
        <h3>Usuarios</h3>
    </div>
</div>
            </header>
            

<div class="row">
    <section>
        <div class="row">
            <div class="small-8 small-offset-2 columns">
<form action="/Account/Register" method="post"><input name="__RequestVerificationToken" type="hidden" value="U7N3EobUC_Rj9kshXSKqorhbr3JTjqA2GoGxXd_j28gU61HK28D3AKBK6Y8PijnOwUygGyLMXA-G8gvVn1-gDTesDM2yqp4XrSM4p298-lo1" /><div class="validation-summary-valid" data-valmsg-summary="true"><ul><li style="display:none"></li>
</ul></div>                    <fieldset>
                        <legend>Creación de nuevo usuario</legend>
                        <div class="row">
                            <div class="small-2 columns">
                                <label for="UserName">Usuario</label>
                            </div>
                            <div class="small-10 columns">
                                <input data-val="true" data-val-required="The Usuario field is required." id="UserName" name="UserName" type="text" value="" />
                            </div>
                        </div>
                        <div class="row">
                            <div class="small-2 columns">
                                <label for="Password">Contrase&#241;a</label>
                            </div>
                            <div class="small-10 columns">
                                <input data-val="true" data-val-length="El Contraseña debe ser al menos de 6 caracteres de largo." data-val-length-max="100" data-val-length-min="6" data-val-required="The Contraseña field is required." id="Password" name="Password" type="password" />
                            </div>
                        </div>
                        <div class="row">
                            <div class="small-2 columns">
                                <label for="ConfirmPassword">Confirmar contrase&#241;a</label>
                            </div>
                            <div class="small-10 columns">
                                <input data-val="true" data-val-equalto="Las contraseñas no coinciden." data-val-equalto-other="*.Password" id="ConfirmPassword" name="ConfirmPassword" type="password" />
                            </div>
                        </div>
                        <div class="row">
                            <div class="small-2 small-offset-2">
                                <input type="submit" value="Crear usuario" class="button" />
                            </div>
                        </div>
                    </fieldset>
</form>            </div>
        </div>
    </section>
</div>


        </section>

        <script src="/Scripts/vendor/jquery.js"></script>

        <script src="/Scripts/vendor/jquery.cookie.js"></script>

        <script src="/Scripts/vendor/fastclick.js"></script>

        <script src="/Scripts/vendor/modernizr.js"></script>

        <script src="/Scripts/vendor/placeholder.js"></script>

        <script src="/Scripts/foundation.min.js"></script>

        <script src="/Scripts/Application.js"></script>
;
        
    <script src="/Scripts/jquery.unobtrusive-ajax.js"></script>
<script src="/Scripts/jquery.validate.js"></script>
<script src="/Scripts/jquery.validate.unobtrusive.js"></script>


    </body>
</html>
